### ISSUE TITLE
Make center/zoom props optional when bounds are provided

### ISSUE THREAD

**Issue by Horrih** [vue-leaflet/vue-leaflet#294]
Hello,
When migrating from vue2 to vue3, we encountered a discrepancy :
- In vue2-leaflet, if you provide bounds, setting the center/zoom is optional since both settings are exclusive
- In vue3 with vue-leaflet, you have to provide a dummy center + zoom even if it gets overridden right away by the bounds.

Plus you have to be extra careful in the order of declaration of props : you have to declare the bounds prop last otherwise it will be overridden by the center.

Here is what we did eventually :

` <l-map :zoom="1" :center="[0, 0]" :bounds="bounds">`

**Comment by fgerards** [comment]
indeed, its VERY odd that zoom isnt optional. this completely undermines the use of bounds and is not intended by leaflet library

**Comment by fgerards** [comment]
btw, what does your "bounds" object look like ?

**Comment by Horrih** [comment]
Basically our code looks like this

HTML section

    <l-map :zoom="zoom" :center="[0, 0]" :bounds="bounds">

TS Section

       import { latLngBounds } from "leaflet"
       const zoom = ref(1)
       const bounds = computed(() => {
           return latLngBounds([
               [lat1, long1],
               [lat2, long2]
           ]))
       })
